cwltool/command_line_tool.py

```diff
@@ -83,7 +84,7 @@
 
 ACCEPTLIST_EN_STRICT_RE = re.compile(r"^[a-zA-Z0-9._+-]+$")
 ACCEPTLIST_EN_RELAXED_RE = re.compile(r".*")  # Accept anything
```

Comment:
```suggestion
ACCEPTLIST_EN_STRICT_RE = re.compile("^[\w0-9._+\- \u2600-\u26FF]+$")  # accept unicode word characters and emojis
ACCEPTLIST_EN_RELAXED_RE = re.compile(r".*")  # Accept anything
```
Better to fix this here, I think?

Reply:
I had it like that at first, but thought the `EN` implied english and this is decidedly not english :smile: 

I tried tox but that exits on the first failure (Maybe because of --strict for pytest, but I tried removing that with no luck). `test_udocker_should_display_memory_usage` fails because psutil removed travis support in version 5.8.0 and I can't install the pinned version on my system. I pinned that version of psutil, as that might get it to work on others' systems.

Removing the `TRAVIS` tests and running with tox works normally, as does invoking `pytest` manually, but there's many failures like e.g. 
```
_________________________________________________ test_dont_require_inputs __________________________________________________

    def test_dont_require_inputs():
        if sys.version_info[0] < 3:
            stream = BytesIO()
        else:
            stream = StringIO()

        script = None
        try:
>           script = NamedTemporaryFile(mode="w", delete=False)

../../tests/test_toolargparse.py:134:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/home/zenon/.pyenv/versions/3.6.12/lib/python3.6/tempfile.py:551: in NamedTemporaryFile
    (fd, name) = _mkstemp_inner(dir, prefix, suffix, flags, output_type)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

dir = '/home/zenon/DEV/ETH/cwltool/tmp/test_cid_file_dir___parallel__0/tmpaip03ivw', pre = 'tmp', suf = '', flags = 131266
output_type = <class 'str'>

    def _mkstemp_inner(dir, pre, suf, flags, output_type):
        """Code common to mkstemp, TemporaryFile, and NamedTemporaryFile."""

        names = _get_candidate_names()
        if output_type is bytes:
            names = map(_os.fsencode, names)

        for seq in range(TMP_MAX):
            name = next(names)
            file = _os.path.join(dir, pre + name + suf)
            try:
>               fd = _os.open(file, flags, 0o600)
E               FileNotFoundError: [Errno 2] No such file or directory: '/home/zenon/DEV/ETH/cwltool/tmp/test_cid_file_dir___parallel__0/tmpaip03ivw/tmpkssk2xd5'

/home/zenon/.pyenv/versions/3.6.12/lib/python3.6/tempfile.py:262: FileNotFoundError
```

but then 
```
$ pytest -k test_dont_require_inputs
==================================================== test session starts ====================================================
platform linux -- Python 3.6.12, pytest-4.2.1, py-1.10.0, pluggy-0.13.1
rootdir: /home/zenon/DEV/ETH/cwltool, inifile: setup.cfg
plugins: mock-3.5.1, cov-2.11.1
collected 392 items / 391 deselected / 1 selected

tests/test_toolargparse.py .                                                                                          [100%]

========================================= 1 passed, 391 deselected in 2.62 seconds ==============================
```

So I'm not sure which failures are real failures.